src/data/padics/padic_norm.lean

```diff
@@ -297,6 +317,152 @@ theorem min_le_padic_val_rat_add {q r : ℚ}
 
 end padic_val_rat
 
+section padic_val_nat
+
+/--
+A rewrite lemma for `padic_val_nat p (q * r)` with conditions `q ≠ 0`, `r ≠ 0`.
+-/
+protected lemma mul (p : ℕ) [p_prime : fact p.prime] {q r : ℕ} (hq : q ≠ 0) (hr : r ≠ 0) :
```

Comment:
What namespace is this now in?
Anyway, this name doesn't seem to follow the same convention as the one below it.

Reply:
This isn't ready to merge - this PR contains https://github.com/leanprover-community/mathlib/pull/3230 merged into it.

Reply:
Ooh, right. Sorry I lost track